controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -101,6 +101,9 @@ func (info *reconcileInfo) publishEvent(reason, message string) {
 // +kubebuilder:rbac:groups=metal3.io,resources=dataimages,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups=metal3.io,resources=dataimages/status,verbs=get;update;patch
 
+// Allow for updating hostupdatepolicies
+//+kubebuilder:rbac:groups=metal3.io,resources=hostupdatepolicies,verbs=get;list;watch;create;update;patch
```

Comment:
I don't think BMO will ever edit these resources? I think having get,list,watch should be enough.

Reply:
Good point. I adjusted this and force-pushed. Thank you!